### PR TITLE
Hotfix, make prometheus storage size configurable

### PR DESCRIPTION
**Pull request by myaser**
make prometheus storage size configurable

### CODE REVIEW

**Comment by demonCoder95** [comment]
👍

**Comment by demonCoder95** [comment]
Not sure if it really matters but maybe put the value inside quotes `"50Gi"` instead of `50Gi`.

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -356,6 +356,7 @@ prometheus_mem_min: "2Gi"
 prometheus_cpu_min: "0"
 prometheus_retention_size: "49GB" # one GB less than prometheus' PVC
 prometheus_retention_time: "1d"
+prometheus_storage_size: 50Gi
```
Comment (myaser) [inline cluster/config-defaults.yaml]:
```suggestion
prometheus_storage_size: "50Gi"
```

**Comment by demonCoder95** [comment]
👍

**Comment by myaser** [comment]
👍🏻

**Pull request merged**